bigchaindb/commands/bigchain.py

```diff
@@ -146,7 +146,13 @@ def run_init(args):
 def run_drop(args):
     """Drop the database"""
     bigchaindb.config_utils.autoconfigure(filename=args.config, force=True)
-    db.drop(assume_yes=args.yes)
+    try:
+        db_name = 'Unknown database'
+        if db != None: db_name = db.get_database_name()
```

Comment:
`db` is defined in [line 22](https://github.com/mickhalsband/bigchaindb/blob/32fcbc0bbf2a8965aec160b42c8194d2737fad8c/bigchaindb/commands/bigchain.py#L22) (it's an import) so it cannot be `None`.

The name of the database is defined even if there is no configuration available (see [`bigchaindb/__init__.py`](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/__init__.py#L20)), so you can assume `db.get_database_name()` to always return a value.